### DISCUSSION TITLE
Enemy Possesed Movement?

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#391] [Q&A] [answered]
How can i move when i posessed an enemy?
WASD doesn't seem to work :(((

**Comment by winstxnhdw** [comment]
Yeap, it’s WASD.

**Comment by Totoqoe** [comment]
It's supposed to be WASD but it's currently bugged, you can use noclip with N as a workaround

**Reply by winstxnhdw** [reply]
It’s bugged on latest?

**Reply by Totoqoe** [reply]
It seems so, you get stuck unless you enable noclip, AstroBoys enemy branch doesn't have this issue except for the masked.

**Reply by winstxnhdw** [reply]
Christ. I hate how unstable possession is.

**Reply by Dexteriority** [reply]
I mean it's all good if we can't control the enemy attacks behavior. but at least we can move it🤔

**Reply by Dexteriority** [reply]
Like the first one, we can only move them, it's fun anyway since the enemies has passive skill whenever they touched a player

**Comment by winstxnhdw** [comment, marked as answer]
Ok, this is fixed in latest. I was refactoring @LTYGUY's code and did not see the conditional closely.

**Reply by Dexteriority** [reply]
Thank you!

**Reply by Dexteriority** [reply]
By the way, i just tested it out, After i press **=** and went to posses the enemy with **M3**.
I somehow still in FreeCam mode, and the WASD controlling both my FreeCam and the Enemy i possesed.
Like the camera is not locking behind the Enemy like usual :((

**Reply by winstxnhdw** [reply]
Some enemies don’t lock. Like bees and spiders IIRC.

**Reply by Dexteriority** [reply]
I tried it out on Eyeless Dog and Forest Giant.

**Reply by winstxnhdw** [reply]
dammit, i only tested on bees so i didn’t catch this behaviour. i’ll look into it tomorrow.

**Reply by Dexteriority** [reply]
Thanks!

**Reply by Dexteriority** [reply]
This is the only issue we have for now, i will be reporting any glitch i found with lc-hax :D
